src/main/java/org/jboss/query/SearchQuery.java

```diff
@@ -35,10 +35,10 @@ private SearchQuery(List<IssueStatus> statuses, String assignee, List<String> pr
         this.labels = labels;
 
         if (startDate != null && startDate.isAfter(LocalDate.now()))
-            throw new IllegalArgumentException("startDate cannot be in the future.");
+            throw new IllegalArgumentException("after cannot be in the future.");
```

Comment:
```suggestion
            throw new IllegalArgumentException("after parameter cannot be in the future.");
```

just to make it clear what you mean

Reply:
The `after` and `before` parameters are used for retrieving issues after a certain time or before respectively. Do you have anything better on mind? I will try to make the message more explicit